Dockerfile

```diff
@@ -1,4 +1 @@
 FROM jupyter/scipy-notebook
```

Comment:
this Dockerfile is officially irrelevant
we should just pull directly from `jupyter/scipy-notebook`